install_config/install/prerequisites.adoc

```diff
@@ -366,9 +366,11 @@ advanced installation of {product-title} (using Ansible), because this option
 overrides the default IP address set by xref:../../admin_solutions/master_node_config.adoc#node-config-options[`*dnsIP*`].
 
 Instead, allow the installer to configure each node to use *dnsmasq* and forward
-requests to SkyDNS or the external DNS provider. If you do set the
-`*openshift_dns_ip*` option, then it should be set either with a DNS IP that
-queries SkyDNS first, or to the SkyDNS service or endpoint IP (the Kubernetes
+requests to SkyDNS, the internal DNS service that provides cluster-wide DNS
+resolution of internal hostnames for services and pods, or the external DNS
+provider. If you do set the
```

Comment:
I'm worried the new information has made the "or the external DNS provider" bit a little vague on what it's modifying. Maybe brackets? So:

"Instead, allow the installer to configure each node to use *dnsmasq* and forward request to SkyDNS (the internal DNS service for cluster-wide DNS resolution of internal hostnames for services and pods) or the external DNS provider."

Just a suggestion. Breaks up the sentence at least?

Reply:
IBM style says to avoid parentheses, but that's a good point. How does swapping the order of the DNS types sound to you?

"Instead, allow the installer to configure each node to use dnsmasq and forward request to the external DNS provider or SkyDNS, the internal DNS service for cluster-wide DNS resolution of internal hostnames for services and pods."

Reply:
@kalexand-rh Sounds good to me!